docs/release_notes/v1.9.5.md

```diff
@@ -0,0 +1,19 @@
+# Dapr 1.9.5
+
+### Fixes an issue in the CosmosDB state store component when performing transaction request with ETags
+
+#### Problem
+
+When setting any ETag in a transaction request inside CosmosDB, dapr exits due to a nil pointer dereference.
+
+### Impact
+
+This issue impacts all users of CosmosDB state store.
+
+#### Root cause
+
+When setting an ETag on a transaction request in CosmosDB state store, a nil pointer was being dereferenced. 
+
+#### Solution
+
+We have changed the CosmosDB statestore code to create a new instance of the 'azcosmos.TransactionalBatchItemOptions' struct and we now use the address of that structure for the ETag.
```

Comment:
That might be too specific, but I bet nobody reads this anyway :D. I find it entertaining, so fine to leave by me

Reply:
I agree with Bernd this is wayyyyy too into the details and wouldn't mean much to 99.9% of Dapr users.

Please just change to "We fixed the bug that caused Dapr to panic."